test/correctness/checker.rs

```diff
@@ -36,8 +37,9 @@ fn main() -> Result<(), Error> {
     LogBuilder::from_env(Env::new().filter_or("MZ_LOG_FILTER", "info"))
         .target(Target::Stdout)
         .init();
-
-    mz_process_collector::register_default_process_collector()?;
+    // TODO: is this even necessary? We don't seem to be reporting them from this process
+    let metrics_registry = MetricsRegistry::new();
+    mz_process_collector::register_default_process_collector(&metrics_registry);
```

Comment:
Oh yeah, just ditch this. This was derived from peeker so almost certainly just got cargo culled.